native/machine-coding/bmi-calculator/script.js

```diff
@@ -1,15 +1,40 @@
-function calculateBMI() {
-    var height = parseFloat(document.getElementById("height").value);
-    var weight = parseFloat(document.getElementById("weight").value);
-  
-    if (isNaN(height) || isNaN(weight)) {
-      alert("Please enter valid height and weight.");
-      return;
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I think `DOMContentLoaded` is not required because the script is defered. Please check if everything works fine without `DOMContentLoaded`. If yes then it can be removed.

Reply:
Ok i fixed it